taskcluster/ios-package.sh

```diff
@@ -16,13 +16,13 @@ package_libdeepspeech_as_zip "libdeepspeech.zip"
 
 case $arch in
 "--x86_64")
-  ${TAR} -cf - \
-         -C ${DS_ROOT_TASK}/DeepSpeech/ds/native_client/swift/DerivedData/Build/Products/Release-iphonesimulator/ deepspeech_ios.framework \
-         | ${XZ} > "${TASKCLUSTER_ARTIFACTS}/deepspeech_ios.framework.x86_64.tar.xz"
+  artifact_name="deepspeech_ios.framework.x86_64.tar.xz"
   ;;
 "--arm64")
-  ${TAR} -cf - \
-         -C ${DS_ROOT_TASK}/DeepSpeech/ds/native_client/swift/DerivedData/Build/Products/Release-iphoneos/ deepspeech_ios.framework \
-         | ${XZ} > "${TASKCLUSTER_ARTIFACTS}/deepspeech_ios.framework.arm64.tar.xz"
+  artifact_name="deepspeech_ios.framework.arm64.tar.xz"
 ;;
 esac
+
+${TAR} -cf - \
+       -C ${DS_ROOT_TASK}/DeepSpeech/ds/native_client/swift/DerivedData/Build/Products/Release-iphonesimulator/ deepspeech_ios.framework \
```

Comment:
Aren't you going to have an issue here with `Release-iphonesimulator` ? ARM64 builds seems to end up in `Release-iphoneos` ?

Reply:
ugh. nice catch